CHANGELOG.md

```diff
@@ -4,9 +4,13 @@
 
 ### Removed
 * Removed mentions of Dash events in OncoPrint component.
+* Removed properties which weren't used in Ideogram component.
 
 ### Changed
-* Changed property `fullChromosomeLabels` so that it can be updated using dash callbacks
+* Changed property `fullChromosomeLabels` in Ideogram component so that it can be updated using 
+dash callbacks.
+* Restricted possible values for certain properties in Ideogram 
+component to a finite set of values.
```

Comment:
It's not clear to me why lines wrap at such different lengths...? Line 10 vs line 12.